tutorials/persistance-comparison-series/BinaryWriterReader/binary_reader_writer.md

```diff
@@ -0,0 +1,426 @@
+# Saving Data in Unity3D Using BinaryReader and BinaryWriter
+<b>(Part 3 of the Persistence Comparison Series)</b>
+
+## Introduction
+
+Persisting data is an important part of most games. Unity offers only a limited set of solutions, which means we have to look around for other options as well.
+
+In Part 1 of this series, we explored Unity's own solution: `PlayerPrefs`. This time, we look into one of the ways we can use the underlying .NET framework by saving files. Here is an overview of the complete series:
```

Comment:
This introduction is the same as the one for Part2. I suppose you want to change it?

Reply:
Yes, it is. I copied one or two paragraph (part from describing the example itself) from the previous article. This is one of them. I've fixed that in ac15b8e9decad9f95666d5e37b811f651dc4a7dc.